components/admin_console/billing/billing_subscriptions/index.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
You should be using hooks to connect `redux` to this component. Adding a connector to a functional component is an anti-pattern as far as I know.

I'm cool with using a connector but then we should convert the component to a class component, which is also fine.

Reply:
+1 to Devin's point. Recommend using hooks and getting rid of this connector component.

Reply:
I've converted it to using hooks, let me know what you think.